0001-Fixes.patch

```diff
@@ -0,0 +1,150 @@
+From 4d0a9a1a82d93b241810446bf0b19eb9d9793f2b Mon Sep 17 00:00:00 2001
```

Comment:
What is this file for?

Reply:
This was included by mistake. I'll remove it.